src/status_im/common/validators.cljs

```diff
@@ -13,3 +13,11 @@
   (and (string? s)
        (not-empty s)
        (boolean (re-matches constants/regx-compressed-key s))))
+
```

Comment:
hmm why do we have common/validators 
and common/validation?
probably should just be one name space properly organised.
cc @mohsen-ghafouri wdyt?

Reply:
Hmm.. I'll look into it and see if they can be moved over to one file

Reply:
Personally I don't think we should move to one file, rather one folder with various uses.

E.g
Common/validation/profile

Common/validation/wallet

Reply:
it's good idea to move common regex to different file as validators and also have sperate validation file for different use case as you mention like `validation/profile`

Reply:
moved the ones under `validators` to a common namespace: `common/validation/general`